docs: initiate rendered documentation via ndg (#43)
feel-co/hjem: CONTRIBUTING.md

# Contribution Guidelines

## Table of Contents

- [Preface](#preface)
- [Contributing Process](#contributing)
- [Code of Conduct](#code-of-conduct)

## Preface

[LICENSE]: ../LICENSE

> We are glad you are thinking about contributing to Hjem! The project is large

Reviewer: ```suggestion
We are glad you are thinking about contributing to Hjem! The project is largely
```

shaped by contributors and user feedback, and all contributions are appreciated.

If you are unsure about anything, whether a change is necessary or if it would

> be accepted _had_ you created a PR, please just ask! Or submit the issue or pull

Reviewer: ```suggestion
be accepted _were_ you to create a PR, please just ask! Or, submit the issue or pull
```

line. A commit message ideally, but not necessarily, follow the following
template:

```txt
{component}: {description}

{long description}
```

- `component` refers to the module or file you are editing.
- `description` is a short description of your change
- `long description` is the optional addition that should be appended if the
  short description cannot sufficiently convey the motive for the change

In rare cases where a PR affects multiple unrelated components, then the
`component` part can be replaced with a generic scope such as `treewide` or
`various.`

### Code of Conduct

Hjem does not have a formal Code of Conduct yet, and we are sincerely hoping
that we will ever need one. This project is not expected to be a hotbed of
activity, and you should be perfectly capable of keeping it civil and
respectful.

That said, everyone who partakes around Hjem and Hjem-adjacent communities or
contributes to Hjem must be allowed to feel welcome and safe. As such, any
parties that disrupt the project or engage in negative behaviour will be dealt
with swiftly and appropriately. You are invited to share any concerns that you

> have with the projects moderation, be it over public or public spaces. T

Reviewer: was the T supposed to be something or is it just a mistake?

```suggestion
have with the projects moderation, be it over public or public spaces.
```
